views/component/courseModules/template.njk

```diff
@@ -0,0 +1,27 @@
+<ul class="list module-list">
+    {% for module in modules %}
+        <li class="module-item list-item u--border--bgrey">
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    <h3 class="govuk-heading-s">{{ module.title }}</h3>
+
+                    <p class="govuk-body">{{ module.description }}</p>
+
+                    {% if module.optional === true %}
+                        <p class="optional">This module is optional</p>
+                    {% endif %}
+
+                    <a href="#" class="govuk-body govuk-link">Start</a>
+                </div>
+                <div class="govuk-grid-column-one-third">
+                    <p class="govuk-body type">{{ module.type }}</p>
```

Comment:
You should be passing to i18 to make sure your types are human readable

e.g. face-to-face > Face-to-face

Reply:
Fixed